frontend/src/components/Admin/DevPortfolio/AdminDevPortfolio.tsx

```diff
@@ -37,10 +24,35 @@ const AdminDevPortfolio: React.FC = () => {
     getAllDevPortfolios();
   }, []);
 
+  const fullReset = () => {
+    setIsLoading(true);
+    setDevPortfolios([]);
+  };
+
+  useEffect(() => {
+    const cb = () => {
+      fullReset();
+    };
+    Emitters.devPortfolioUpdated.subscribe(cb);
```

Comment:
I think you can just pass `fullReset` directly in here. No need for the extra callback function.